docs/ensnode.io/src/styles/starlight.css

```
@@ -308,13 +307,36 @@ a[rel="prev"]:hover {
+#starlight__sidebar details > summary:hover {
   background-color: rgba(128, 128, 128, 0.1);
   transform: translateX(2px);
   border-radius: 6px;
 }
 
+/** Additional styling required for the
+ * expandable items in the sidebar
+ */
+#starlight__sidebar details details > summary:hover {
+  transition: all 0.2s ease-in-out;
+  color: var(--sl-color-gray-2);
```

Stylelint (17.11.1)

[error] Duplicate property "color" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)